datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dataset/mappers/DatasetMapper.java

```diff
@@ -136,6 +137,20 @@ private void mapDatasetProperties(@Nonnull Dataset dataset, @Nonnull DataMap dat
         properties.setQualifiedName(gmsProperties.getQualifiedName());
         dataset.setProperties(properties);
         dataset.setDescription(properties.getDescription());
+        TimeStamp created = gmsProperties.getCreated();
+        if (created != null) {
```

Comment:
This looks good! Would it be possible to add some tests here to make sure we don't get any NPE's?

Reply:
Added test case to cover if any NPE.